tests/test_view_model.py

```diff
@@ -0,0 +1,115 @@
+from datetime import datetime, timedelta
+from app.view_model import ViewModel
+from app.item import Item
```

Comment:
This file actually appears to be at the root folder so this import is failing (here and elsewhere); I'm guessing the intention was to move the file into the app folder?